.github/workflows/update-ims.js

```diff
@@ -0,0 +1,114 @@
+const https = require('https');
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+const fetchImsScript = () =>
+  new Promise((resolve, reject) => {
+    https
+      .get('https://auth.services.adobe.com/imslib/imslib.min.js', (res) => {
+        if (res.statusCode < 200 || res.statusCode >= 300) {
+          return reject(new Error('statusCode=' + res.statusCode));
+        }
+
+        let data = '';
+
+        res.on('data', (chunk) => {
+          data += chunk;
+        });
+
+        res.on('end', () => {
+          resolve({
+            data,
+            headers: res.headers,
+          });
+        });
+      })
+      .on('error', (err) => {
+        reject(err);
+      });
+  });
+
+const main = async ({ github, context, local = false }) => {
+  const response = await fetchImsScript();
+  const lastModified = new Date(response.headers['last-modified']);
+  const now = new Date();
+  const diffHours = Math.ceil((now - lastModified) / 1000 / 60 / 60);
+  console.log(
+    `imslib.min.js last modified: ${lastModified}. Last modified ${diffHours} hours ago <= 24H, means new PR created.`
+  );
+
+  if (diffHours <= 24 || local) {
+    console.log('Diff detected in the last 24h, creating PR.');
+    const header = `// Last modified ${lastModified.toISOString()}\n// Build at ${new Date().toISOString()}\n`;
+
+    if (local) return console.log('Local mode - skipped git commands.');
```

Comment:
Could this not be the first declaration to skip the previous console log and header declaration?